docker/index.js

```diff
@@ -13,7 +13,7 @@ const { parsePath, formatPath } = require('./util');
 const app = express();
 
 const port = 80;
-const blockchainEndpoint = process.env.PROVIDER_URL;
+const blockchainEndpoint = process.env.TESTNET_PROVIDER_URL;
```

Comment:
Why didn't you unify it with PROVIDER_URL?

Reply:
may be.. TESTNET_PROVIDER_URL have to be worte if use ainize. so i separate PROVIDER_URL and TESTNET_PROVIDER_URL.

Reply:
Can I deploy to ainize dev and ainizve prod at this time?

Reply:
@dlgochan Q) what is difference between ainize dev and ainize prod ?

Reply:
[ainize dev](https://ainize-dev.herokuapp.com/dashboard)
[ainize](https://ainize.ai/)

The difference between the two is whether the PROVIDER_URL is set to mainet or testnet when deploying.

Reply:
but now, I don't know both are available.. (Due to ainize operation suspension issue)

Reply:
oh... because we used ainize prod to deploy this app, we need to declare TESTNET_PROVIDER_URL variable for testing. however if ainize-dev is still available, there are no reason to use "process.env.TESTNET_PROVIDER_URL" anymore. this issue will be changed after testing on ainize-dev.

Reply:
I'll check if ainize dev works and let you know.
I think I need to use ainize-dev and prod separately even for the aintfize test environment.